hugegraph-dist/src/main/java/com/baidu/hugegraph/dist/HugeGremlinServer.java

```diff
@@ -33,33 +33,42 @@
 
     private static final String G_PREFIX = "__g_";
 
-    public static void main(String[] args) throws Exception {
+    static {
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            LOG.info("HugeGremlinServer stopped");
+        }));
+    }
+
+    private static ContextGremlinServer server;
```

Comment:
prefer to let HugeGremlinServer.start() return an instance, same with HugeRestServer.start()